### DISCUSSION TITLE
Will lotus ever be able to download client data on its own?

### DISCUSSION THREAD

**Discussion by Shekelme** [filecoin-project/community#539] [General] [answered]
Will `lotus` ever learn how to download client data on its own? I observe 122 downloaded files from different clients and almost zero download speed. No restarts help to get things off the ground. Sure, there is `boost`, but...

**Comment by Stefaan-V** [comment]
Can you elaborate a little bit more on your setup and results please?

Are you currently running BOOST?

**Comment by Shekelme** [comment]
Two miners, one 200 mbit channel. Despite the limitations in the miner's config, the download list swells to 80-100 or more download items, and downloads stop from time to time. And to such an extent that restarting the miner does not help. I don't use `boost` yet, pure `lotus`. One monolyth miner and another with separated markets process.

**Comment by rjan90** [comment, marked as answer]
Lotus knows how to download data from a client, but as you note there can be multiple things that go wrong, which causes a `data-transfer` to cancel out. Just restarting the transfer might not fix the issues you are seeing because it might be that the client is not available anymore.

A lot of imporvements are also being made in the underlying transports and dependencies, `go-fil-markets` and `go-data-transfer`, that gets implemented in Lotus.

But as noted in the latest release notes for Lotus: _Markets related features, enhancements and fixes is now lower priority for Lotus, and is going to be in the hands of [Boost](https://boost.filecoin.io/), built by the amazing [Bedrock team](https://www.notion.so/pl-strflt/Bedrock-2e956d5d8143432080a1d84435ccf0ff)._

It does not mean that Lotus does not care about storage deals / retrievals, but it will not be the focus to develop that within the monolith `lotus-miner`. The overall theme that Lotus is focusing on in Q3/Q4, is scalability for `lotus-miner` for sealing and proving, as well as supporting features that are needed in `lotus-miner` to enable Boost to let SPs onboard and serve a lot of data.

**Comment by Shekelme** [comment]
Alas, this was expected.

**Comment by jennijuju** [comment]
@Shekelme this is a good q! im curious about:
- have you tried boost yet?
- if so, does it resolve the issue?
- If not, why? Any blocker or concern?
